app/controllers/sitemaps_controller.rb

```diff
@@ -41,10 +41,14 @@ def general
       [blog_posts_url, BlogPost.published.pluck(:updated_at).last, :monthly, 0.95]
     ]
 
-    BlogPost.published.ordered_by_recency.each do |blog_post|
+    BlogPost.published.ordered_by_recency.find_each do |blog_post|
       pages << [blog_post_url(blog_post), blog_post.updated_at, :monthly, 0.75]
     end
 
+    Document.where(track: nil).find_each do |doc|
+      pages << [doc_url(doc.section, doc.slug), doc.updated_at, :monthly, 0.8]
```

Comment:
Let me know what the important should be

Reply:
`0.75`